src/components/Trade/hooks/useSwapper/useSwapper.tsx

```diff
@@ -97,8 +97,15 @@ export const useSwapper = () => {
     selectBIP44ParamsByAccountId(state, sellAccountFilter),
   )
 
+  /*
+  Cross-account trading means trades that are either:
+    - Trades between assets on the same chain but different accounts
+    - Trades between assets on different chains (and possibly different accounts)
+
+   When adding a new swapper, ensure that `true` is returned here if either of the above apply.
+   */
   const swapperSupportsCrossAccountTrade = useMemo(() => {
-    if (!bestTradeSwapper) return false
+    if (!bestTradeSwapper) return undefined
```

Comment:
No runtime changes here, but semantically more correct - if we have no swapper, the disposition of a swapper can be neither `true` or `false`.